packages/jest-worker/src/workers/processChild.ts

```diff
@@ -34,7 +34,7 @@ let initialized = false;
  * If an invalid message is detected, the child will exit (by throwing) with a
  * non-zero exit code.
  */
-process.on('message', (request: any) => {
+const messageListener = (request: any) => {
```

Comment:
`MessageListener` is typed in DT, but I don't know if we can access it?

https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b8183c0147e7412a4e0414a5456441789473b4d8/types/node/globals.d.ts#L726

Reply:
Can perhaps use `Parameters<>` to extract it somehow, but probably so ugly that it's not worth it. The params of the listener are still typechecked this way when passed to `process.on`

Reply:
yeah, agreed